web/skins/classic/js/skin.js

```diff
@@ -283,6 +283,17 @@ function convertLabelFormat(LabelFormat, monitorName){
 }
 
 function addVideoTimingTrack(video, LabelFormat, monitorName, duration, startTime){
+//This is a hacky way to handle changing the texttrack. If we ever upgrade vjs in a revamp replace this.  Old method preserved because it's the right way.
+  let cues = vid.textTracks()[0].cues();
```

Comment:
Feel free to update video.js.  The only reason we havn't is that it is low on our todo lists. It is slightly non-trivial.